core/domain/summary_services.py

```diff
@@ -27,8 +27,14 @@ def get_human_readable_contributors_summary(contributors_summary):
     contributor_ids = contributors_summary.keys()
     contributor_usernames = user_services.get_human_readable_user_ids(
         contributor_ids)
+    contributor_profile_pictures = \
+        user_services.get_profile_pictures_by_user_ids(contributor_ids)
     return {
-        contributor_usernames[ind]: contributors_summary[contributor_ids[ind]]
+        contributor_usernames[ind]: {
```

Comment:
Can you add a test along the following lines: create an exploration with two contributors -- one of them has a profile picture and makes the initial commit, and the other doesn't have a profile picture and makes two subsequent commits. Then get its summary. The contributor_usernames field returned here should be correct.